### DISCUSSION TITLE
Parallel Routes: expected behavior in Vercel "App Router Playground" on hard refresh / direct link?

### DISCUSSION THREAD

**Discussion by scnroy** [vercel/next.js#59675] [App Router]
Hi all,

When visiting these links (or refreshing in the browser), I see UI from `default.tsx` rendered instead of the UI from each of the parallel routes:
https://app-router.vercel.app/parallel-routes/view-duration
https://app-router.vercel.app/parallel-routes/impressions

After i navigate back 'home', the UI appears as expected. i'm also experiencing this same issue in my own app (hard refreshes don't restore app state) and was hoping i had some detail wrong in my implementation, but maybe this is how it is supposed to work? Either way, I have found it confusing.

Would love to hear from others what the expected behavior is for urls that match parallel routes and whether or not this is a bug.

Thanks!

**Comment by jasperhartong** [comment]
I'm running into this same issue. For now I've resolved it by re-exporting my Home in `default.tsx`:
```javascript
import Home from "./page";
// Re-export Home in default to ensure direct navigation to modal-routes will render the website underneath (e.g. /en/contact)
export default Home;
```

Seems to be working, but I would assume this to be the default behaviour for the children slot. Also not sure about any side-effects with this re-export.
